apps/expo/src/app/calendar.tsx

```diff
@@ -1,13 +1,79 @@
 import React from "react";
-import { Text } from "react-native";
+import { Image, Text, View } from "react-native";
+import type { ImageSourcePropType } from "react-native";
 import { SafeAreaView } from "react-native-safe-area-context";
 import { Stack } from "expo-router";
 
+import EventTab from "~/components/EventTab";
+import Logo from "../../assets/logo.png";
+import TabNav from "../components/TabNav/TabNav";
+
 const Calendar = () => {
   return (
-    <SafeAreaView className="">
+    <SafeAreaView className="flex-1 items-center px-4">
       <Stack.Screen options={{ title: "Calendar", headerShown: false }} />
-      <Text>Calendar</Text>
+
+      {/* logo */}
+
+      <Image
+        source={Logo as ImageSourcePropType}
```

Comment:
`declare module '*.png' {
  const value: import('react-native').ImageSourcePropType;
  export default value;
}`

try adding this to index.d.ts so that you don't need to do the typecast

Reply:
Will do, Ryan did the typecast so I copied him. Also that gives me error so I did this
```
declare module "*.png" {
  const value: ImageSourcePropType;
  export default value;
}
```